assets/tests/components/propertiesPanel/minischedule.test.tsx

```diff
@@ -346,6 +346,113 @@ describe("MinischeduleRun", () => {
 
     expect(tree).toMatchSnapshot()
   })
+
+  test("renders duty details of run", () => {
```

Comment:
This shows up in other snapshots, so unless you're testing a specific case here, I don't think having another test for it is necessary.

Reply:
I did specifically want to test a complex case here, to make sure I was exercising all the different parts of the logic.